.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@samuelraeburn, can we make git/sphinx/whatever stop creating this file every time I edit files in these repos?

Let me know if you want me to delete it now. (Or feel free to do so yourself and approve.)

Reply:
Hi Rachel,
I'm not sure why the `.vscode` directory would be added by git, but we cannot include it in this commit.
This directory/file contain the configuration settings Visual Studio code.
What did you do for the `git add` and `git commit` operations?

Either way, you need to remove it from the PR

Reply:
I have removed it